### PR TITLE
Restructured roswww webserver

### PR DESCRIPTION
**Pull request by jihoonl**
@130s 
- classfy roswww webserver
- remove ros runtime dependency. No roscore is required.

To execute

> ./roswww/scripts/webserver.py --port 8081 --webpath www

### CODE REVIEW

**Comment by jihoonl** [comment]
@130s bump

**Comment by 130s** [comment]
Yeah, sorry it's been a crazy week. Hopefully I'll look tonight.

File: launch/roswww.launch [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<launch>
+  <arg name="name" default="roswww"/>
+  <arg name="port" default="8085"/> <!-- avoid to use apache default port -->
```
Comment (130s) [inline launch/roswww.launch:3]: Just curious, why using a port different from your commandline example (`8081`)?

File: package.xml [hunk 2]
```diff
@@ -14,7 +14,8 @@
 
   <buildtool_depend>catkin</buildtool_depend>
   <run_depend>rosbridge_server</run_depend>
-  <run_depend>rospy</run_depend>
+  <run_depend>rospack</run_depend>
+  <run_depend>rosgraph</run_depend>
```
Comment (130s) [inline package.xml:18]: Might want to stick to alphabetical order?

**Comment by 130s** [comment]
LGTM, other than minor comments I made, which someone can handle in the future if needed. I was able to do the following:
- rosrun 

```
$ rosrun roswww webserver.py --port 8081 --webpath www
```

And http://localhost:8081/ showed:

```
ROS web server successfully started.
Package List
actionlib
actionlib_msgs
actionlib_tutorials
amcl
angles
:
```
- roslaunch

```
$ roslaunch roswww roswww.launch  
```

And http://localhost:8085/ returns the same as above.

File: launch/roswww.launch (hunk 1 above)
Comment (jihoonl) [inline launch/roswww.launch:3]: ah It was just my whim. I thought 8081 seems not very unique compare to pot 8085.

**Pull request merged**